cli/exchange/service.go

```diff
@@ -563,3 +603,58 @@ func ServiceRemovePolicy(org string, credToUse string, service string, force boo
 func ServiceNewPolicy() {
 	policy.New()
 }
+
+// Get the Kubernetes operator yaml archive from the cluster deployment string and save it to a file
+func SaveOpYamlToFile(sId string, clusterDeployment string, filePath string) {
+	// get message printer
+	msgPrinter := i18n.GetMessagePrinter()
+
+	archiveData := GetOpYamlArchiveFromClusterDepl(clusterDeployment)
+
+	fileName := filePath
+	if info, err := os.Stat(filePath); err == nil && info.IsDir() {
+		_, id := cliutils.TrimOrg("", sId) // remove the org part
+		fileName = filepath.Join(filePath, id+"_operator_yaml.tar.gz")
+	}
+
+	if _, err := os.Stat(fileName); err == nil {
+		cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("File %v already exist. Please specify a different file name.", fileName))
```

Comment:
change "already exist" to "already exists"

Reply:
done